### PR TITLE
Move single yaml test suite to yaml test case path

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket:  https://progress.opensuse.org/issues/163301
- Related MR: 
  https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/260   
  https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/263                        
- Needles:  N/A
- Verification run: http://openqa.suse.de/tests/overview?version=15-SP6&distri=sle&build=lemon-suse%2Fos-autoinst-distri-opensuse%23move-single-yaml-to-yam-testcase-path

### CODE REVIEW

File: schedule/yam/test_cases/ssh-x@s390x_zvm.yaml
Comment (jknphy) [inline schedule/yam/test_cases/ssh-x@s390x_zvm.yaml:1]: let's avoid architecture prefixes inside /test_cases/ our test cases should agnostic in that sense.
could you check for more existing occurrences (not added by you or added by you in this PR)  if they could be fixed in this PR? thx.

Comment (lemon-suse) [reply schedule/yam/test_cases/ssh-x@s390x_zvm.yaml:1]: Removed this, it already replaced by schedule/yam/test_cases/ssh-x.yaml.  I will checked the folder, no schedule files with architecture prefix any more.

**Comment by jknphy** [comment]
Could you please move all the vars to job group yaml in all the files moved?

**Comment by lemon-suse** [comment]
> Could you please move all the vars to job group yaml in all the files moved?

Moved vars to job group yaml.

**Pull request merged**